carbon-design-system/carbon
# Fixed types in `FilterableMultiSelect`

Closes #16491

Fixed types with wrong validation.

#### Changelog

**New**

- Added the types in one file to share between `MultiSelect` and `FilterableMultiSelect`

**Changed**

- Change name from `Item` to `ItemType` to match the type in `MultiSelect`

#### Testing / Reviewing

- Make sure the `MultiSelect` and `FilterableMultiSelect` are working as expected
- Make sure types make sense.
